packages/content/src/Application/ContentDataMapper/DataMapper/SeoDataMapper.php

```diff
@@ -35,12 +36,39 @@ public function map(
      */
     private function setSeoData(SeoInterface $dimensionContent, array $data): void
     {
-        $dimensionContent->setSeoTitle($data['seoTitle'] ?? null); // @phpstan-ignore-line TODO where validate this?
-        $dimensionContent->setSeoDescription($data['seoDescription'] ?? null);
-        $dimensionContent->setSeoKeywords($data['seoKeywords'] ?? null);
-        $dimensionContent->setSeoCanonicalUrl($data['seoCanonicalUrl'] ?? null);
-        $dimensionContent->setSeoHideInSitemap($data['seoHideInSitemap'] ?? false);
-        $dimensionContent->setSeoNoFollow($data['seoNoFollow'] ?? false);
-        $dimensionContent->setSeoNoIndex($data['seoNoIndex'] ?? false);
+        if (\array_key_exists('seoTitle', $data)) {
```

Comment:
this was missing. See excerpt as or other data mapper as reference only set data if send else stay untouched.